client/src/components/SignUp.js

```diff
@@ -181,9 +197,10 @@ function SignUp(props) {
           />
           <Button
             className={classes.button}
+            onClick={handleClick}
+            //type="submit"
```

Comment:
I added onSubmit on the form element instead of onClick on the button

Reply:
Looks good!